app/views/SidebarView/index.js

```diff
@@ -114,8 +123,11 @@ class Sidebar extends Component {
 			if	(roles) {
 				const permissionsCollection = db.collections.get('permissions');
 				const permissionsFiltered = await permissionsCollection.query(Q.where('id', Q.oneOf(permissions))).fetch();
-				const isAdmin = permissionsFiltered.reduce((result, permission) => (
-					result || permission.roles.some(r => roles.indexOf(r) !== -1)),
+				const isAdmin = permissionsFiltered.reduce((result, permission) => {
+					this.setState({ isSetAdminNeedCall: false });
```

Comment:
Not a good practice to change the state inside of an iteration.
It's going to set the state for every permissions on the list.
Even if `sCU` prevents the re-render to happen, you're calling unnecessary resources.